src/zero/server.clj

```diff
@@ -44,13 +42,11 @@
   [env]
   (let [environment (env env/stuff)
         {:keys [cookie_secret
-                oauth2_client_id oauth2_client_secret
                 password username]}
         (util/vault-secrets (get-in environment [:server :vault]))
-        result {"COOKIE_SECRET"          cookie_secret
+        result {"WFL_LIVE_SERVER_MODE"   "true"
+                "COOKIE_SECRET"          cookie_secret
```

Comment:
Do we still need a cookie secret?

Reply:
I believe so, the `wrap-defaults` requires `cookie-store` which requires cookie secret